email-templates/src/templates/grant-approved-ticket-travel-accommodation.html

```diff
@@ -0,0 +1,187 @@
+---
+title: "[PyCon Italia 2022] Financial Aid Update!"
+---
+
+<extends src="src/layouts/base.html">
+  <block name="template">
+    <table class="w-full font-sans">
+      <tr>
+        <td align="center" class="bg-gray-100">
+          <table class="w-600 sm:w-full">
+            <tr>
+              <component src="src/components/head-pycon.html" />
+            </tr>
+            <tr>
+              <td align="center" class="sm:px-24">
+                <table class="w-full">
+                  <tr>
+                    <td
+                      class="bg-white text-gray-800 text-base text-left p-48 sm:px-24 leading-24"
+                    >
+                      <p class="m-0 mb-16">👋 @{{firstname}}!</p>
+                      <p class="m-0 mb-16">
+                        Good news! Your grant request for @{{conferenceName}}
+                        has been accepted! 🎉
+                      </p>
+                      <p class="m-0 mb-16">
+                        This means that you have been granted the conference
+                        ticket (includes AM and PM coffee breaks and Lunch for
+                        every conference day) and a bed in a shared room
+                        (available from @{{startDate}} to @{{endDate}}).
+                      </p>
+                      <p class="m-0 mb-16">
+                        Since we had many grant requests we are able to give up
+                        to @{{amount}}€ refund for travel expenses (the amount
+                        granted will be corresponded upon valid payments
+                        receipts, like plane/bus/train tickets, sent to us
+                        before the conference).
+                      </p>
```

Comment:
this copy is a bit confusing

"Since we had many grant requests we are able to give up"

why do you say "Since we had many grants we can give you money for travel"?